.github/workflows/build.yml

```
@@ -724,13 +724,36 @@ jobs:
             && '
             check-autoconf-regen,
             check-generated-files,
+            '
+            || ''
+          }}
+          ${{
+            !fromJSON(needs.build-context.outputs.run-windows-tests)
+            && '
+            build-windows,
+            '
+            || ''
+          }}
+          ${{
+            !fromJSON(needs.build-context.outputs.run-ci-fuzz)
+            && '
+            cifuzz,
+            '
+            || ''
+          }}
+          ${{
+            !fromJSON(needs.build-context.outputs.run-macos)
+            && '
             build-macos,
+            '
+            || ''
+          }}
```

These check a single job, perhaps make them one-liners?
```suggestion
          ${{ !fromJSON(needs.build-context.outputs.run-windows-tests) && 'build-windows,' || '' }}
          ${{ !fromJSON(needs.build-context.outputs.run-ci-fuzz) && 'cifuzz,' || '' }}
          ${{ !fromJSON(needs.build-context.outputs.run-macos) && 'build-macos,' || '' }}
```
Plus for andoid/ios/wasi below?